src/gc.c

```diff
@@ -259,6 +259,20 @@ static int check_timeout = 0;
 static gcpage_t *page_metadata(void *data);
 static void pre_mark(void);
 static void post_mark(arraylist_t *list, int dryrun);
+static region_t *find_region(void *ptr, int maybe);
+jl_taggedvalue_t *jl_gc_find_taggedvalue_pool(char *p, size_t *osize_p);
```

Comment:
For MSVC, you need `DLLEXPORT` to be consistent between the prototype and the implementation. It should either be on both or neither. Which should this be?